packages/eas-cli/src/update/utils.ts

```diff
@@ -41,3 +44,26 @@ export function formatUpdate(update: FormatUpdateParameter): string {
     update.actor as any
   )})`;
 }
+
+export function ensureValidVersions(exp: ExpoConfig, platform: RequestedPlatform): void {
+  const error = new Error(
+    `Couldn't find either 'runtimeVersion' or 'sdkVersion' to configure 'expo-updates'. Please specify at least one of these properties under the 'expo' key in 'app.json'. ${learnMore(
+      'https://docs.expo.dev/eas-update/runtime-versions/'
+    )}`
```

Comment:
Have you verified that the "learn more" link is printed correctly when used in an error message?
If the formatting is preserved then apply my usual comments to `'runtimeVersion'`, `'sdkVersion'`, and `'expo-updates'` :)

Reply:
lol 😄

Reply:
Also verified that the link is printed ( as in I can click on it and it opens a browser window)
<img width="1192" alt="Screen Shot 2022-02-21 at 8 21 39 AM" src="https://user-images.githubusercontent.com/1220444/154963383-8bc9aabc-3663-4a57-9527-6e9a19c98fda.png">